automock/supervisord.conf

```diff
@@ -0,0 +1,16 @@
+[supervisord]
+nodaemon=true
+
+[program:moto]
+command=/home/autopush/pypy/bin/moto_server dynamodb2 -p 5000
```

Comment:
I think we can specify `--config_{shared, connection, endpoint}` as environment variables, and have Supervisor substitute them.